storage/mysql/admin_storage.go

```diff
@@ -355,21 +346,19 @@ func (t *adminTX) CreateTree(ctx context.Context, tree *trillian.Tree) (*trillia
 	insertControlStmt, err := t.tx.Prepare(`
 		INSERT INTO TreeControl(
 			TreeId,
-			ReadOnlyRequests,
 			SigningEnabled,
 			SequencingEnabled,
 			SequenceIntervalSeconds,
 			SignIntervalSeconds)
```

Comment:
SignIntervalSeconds is possibly not used now as IIRC I removed the code that did regular signing. It might still be passed to sequencing in which case this can be addressed separately.